src/ui/renderer.cpp

```diff
@@ -472,11 +473,25 @@ void Renderer::render(Texture& texture,
   int display_height = static_cast<int>(window.display_size().height());
 
   glViewport(0, 0, display_width, display_height);
-  glDisable(GL_BLEND);
+  // Sometimes screencapturelite doing some strange staff with alpha channel
+  // So we usually skip this channel, but it's necesarry if we want to render
+  // Pictures with transparent background, so we have to handle it manually
+  if (render_alpha)
+  {
+    glEnable(GL_BLEND);
+    m_gl.glBlendEquation(GL_FUNC_ADD);
+    glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
+    glEnable(GL_SCISSOR_TEST);
```

Comment:
`GL_SCISSOR_TEST` is not related to alpha channel in any way. This option just allows to crop the render target via `glScissor` call.